app/services/users.js

```diff
@@ -46,3 +46,19 @@ exports.index = async req => {
   });
   return { users: rows, count, page };
 };
+
+const findUserByMail = mail => User.findOne({ where: { mail } });
+
+const invalidateAll = token => {
+  try {
+    return findUserByMail(jwt.validate(token).mail).then(user => {
+      user.tokenEmitDate = Date.now();
+      user.save();
+      return user;
+    });
```

Comment:
Avoid mixing services, also you only need an update query for this and you're doing two queries here